src/index.js

```diff
@@ -1,32 +1,31 @@
 import './styles.css';
+import { update } from './storage.js';
 
-const tasks = [{
-  title: 'first',
-  completed: false,
-  index: 1,
-},
-{
-  title: 'second',
-  completed: false,
-  index: 2,
-},
-{
-  title: 'third',
-  completed: false,
-  index: 3,
-}];
+const tasks = JSON.parse(window.localStorage.getItem('completed'));
 
 const list = document.querySelector('ul');
+
 tasks.forEach((item, i) => {
   const li = document.createElement('li');
   li.className = 'list-item';
   li.innerText = tasks[i].title;
   list.appendChild(li);
   const checkbox = document.createElement('INPUT');
   checkbox.setAttribute('type', 'checkbox');
+  checkbox.checked = tasks[i].completed;
+  checkbox.id = 'checkbox';
   li.appendChild(checkbox);
+  checkbox.addEventListener('change', function () {
+    if (this.checked) {
+      tasks[i].completed = true;
+      update(tasks);
+    } else {
+      tasks[i].completed = false;
+      update(tasks);
+    }
+  });
 });
-
+console.log(tasks);
```

Comment:
- [ ] Avoid committing console.log to your repo. It kills performance and it can make confidential data be visible to anyone using the browser tools to look at your website.